### PR TITLE
Fareye-jwt-auth-in-prod-tag

### PR DESCRIPTION
**Pull request by mborkhat**
Applying JWT auth in  fareye-v1.0.13.3 tag

### CODE REVIEW

File: superset/views/core.py [hunk 1]
```diff
@@ -166,6 +170,17 @@
...
 )
 
+def check_jwt(f):
+    def wraps(self, *args, **kwargs):
+      try:
+          session = current_app.appbuilder.get_session
+          create_session_from_jwt_token()
+          return f(self, *args, **kwargs)
+      except Exception as ex:  # pylint: disable=broad-except
+          logging.warning(ex)
```
Comment (RohitRanjan-GetFareye) [inline superset/views/core.py:180]: use logging.exception

File: superset/additional_authentication_layer.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    session.permanent = True
+    app.permanent_session_lifetime = timedelta(hours=1)
+
+
+def create_session_from_jwt_token():
+   #"Check if url contains mobi_filter"
+   #"If url contains mobi_filter, it validates the user with JWT token and creates a session"
+
+    token = request.args.get("mobi_filter")
+    if token is not None:
```
Comment (RohitRanjan-GetFareye) [inline superset/additional_authentication_layer.py:25]: Just check token is not always returning None.

**Review by RohitRanjan-GetFareye — APPROVED** [review]

File: superset/views/core.py (hunk 1 above)
Comment (mborkhat) [reply superset/views/core.py]: Done changed to logging.exception

File: superset/additional_authentication_layer.py (hunk 2 above)
Comment (mborkhat) [reply superset/additional_authentication_layer.py:25]: Working

**Comment by mborkhat** [comment]
Fareye_prod_version_jwt_implementation

**Pull request merged**